### PR TITLE
Remove dialog host dependencies

### PR DESCRIPTION
**Pull request by SKProCH**
This PR removes a DialogHost dependency, but library still can be installed manually

### CODE REVIEW

File: Material.Styles/DialogHost.axaml [hunk 1]
```diff
@@ -1,17 +0,0 @@
-<Styles xmlns="https://github.com/avaloniaui"
-        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-        xmlns:dialogHost="clr-namespace:DialogHost;assembly=DialogHost.Avalonia"
-        xmlns:styles="clr-namespace:Material.Styles"
-        xmlns:assists="clr-namespace:Material.Styles.Assists">
-    <Style Selector="dialogHost|DialogHost">
```
Comment (appleneko2001) [inline Material.Styles/DialogHost.axaml:6]: Might we keep this fragment and paste it on wiki page or create another package for it?

File: Material.Styles/MaterialToolKit.xaml [hunk 2]
```diff
@@ -20,11 +20,6 @@
     <StyleInclude Source="avares://Material.Styles/SnackbarHost.xaml"/>
     <StyleInclude Source="avares://Material.Styles/MaterialInternalIcon.axaml"/>
```
Comment (appleneko2001) [inline Material.Styles/MaterialToolKit.xaml:22]: leave a comment on this line about dialog host supports is recommended I think

File: Material.Styles/DialogHost.axaml (hunk 1 above)
Comment (SKProCH) [reply Material.Styles/DialogHost.axaml:6]: No, we don't need it. It works fine out of the box with Material. Moreover, I want to create support for Material on the DialogHost.Avalonia side

File: Material.Styles/MaterialToolKit.xaml (hunk 2 above)
Comment (SKProCH) [reply Material.Styles/MaterialToolKit.xaml:22]: Hmm, good idea

Comment (appleneko2001) [reply Material.Styles/MaterialToolKit.xaml:22]: like `<!-- dialog host style have been removed from index of styling for avoid independend package linking -->` or etc.

**Review by appleneko2001 — APPROVED** [review]
Good to go, but I do recommend to read some comments from me and do some changes to leave some important messages for other ppl

File: Material.Styles/DialogHost.axaml (hunk 1 above)
Comment (appleneko2001) [reply Material.Styles/DialogHost.axaml:6]: ok then its good to go

**Review by appleneko2001 — COMMENTED** [review]
LGTM

**Pull request merged**